City-of-Helsinki/drupal-hdbt
# UHF-6847: Filter pills

# [UHF-6847](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6847)
Add filter pills as visual indicator for active filters.

## What was done
- Added shared filter pills component
- Added logic for rendering pills
- 
## How to install

* Set up KASKO instance: https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus
* Install this branch: `composer require drupal/hdbt:dev-UHF-6847-filter-pills`
* This feature can be tested with test server data. Add `ELASTIC_PROXY_URL=https://kasvatus-elastic-proxy-test.agw.arodevtest.hel.fi` to your .env file, then run `make up`
* Run `make drush-cr`

## How to test
Add or edit a landing page to include "School search" paragraph. View the page.

* Navigate to the right-side "tab" of the search application (titled "Find by school features")
* Make some selections using the checkboxes. After submitting, these selectino should be represented as pill buttons at the bottom of the search form.
* Clicking the pills should remove the selection and update the results

## Designers review





[UHF-6847]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-6847?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
